### DISCUSSION TITLE
"goreleaser r --snapshot" but for a single os/arch

### DISCUSSION THREAD

**Discussion by till** [goreleaser/goreleaser#2155] [Ideas, Issue Triage, and General discussions]
I am usually running this a lot when I work on code: `goreleaser release --snapshot --rm-dist`

As projects grow, this tends to take longer and longer, especially when I target different combinations of `GOOS` and `GOARCH`. And most of the time, I want one binary: one for the OS I am working on.

I am wondering what the best way to do this would be. Would anyone "mind" a flag like `--only-current-goos-goarch` ;) (open to a better name).  Or should I maintain a copy of my `.goreleaser.yml` for this purpose? Or maybe an environment variable?

**Comment by caarlos0** [comment]
FWIW you can use `goreleaser build`, and there are a few issues related to "building a specific target" as well

https://github.com/goreleaser/goreleaser/pull/1987
https://github.com/goreleaser/goreleaser/issues/1852

**Reply by caarlos0** [reply]
Resolved on master, will be out on next minor :)
